scripts/uosc/elements/Volume.lua

```diff
@@ -99,7 +99,7 @@ function VolumeSlider:render()
 
 	-- Draws a rectangle with nudge at requested position
 	---@param p number Padding from slider edges.
-	---@param r number Padding from slider edges.
+	---@param r number Border radius.
```

Comment:
Corner radius?

Reply:
`border-radius` is [kind of a convention](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius).

Reply:
Fair enough, but wouldn't that make the edges of the notch also part of the border? :thinking: